config/frontend.ssl.properties

```diff
@@ -0,0 +1,42 @@
+#
+# Copyright (C) 2016 LinkedIn Corp. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+# this file except in compliance with the License. You may obtain a copy of the
+# License at  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied.
+#
+
+# rest server
+rest.server.blob.storage.service.factory=com.github.ambry.frontend.AmbryBlobStorageServiceFactory
+rest.server.router.factory=com.github.ambry.router.NonBlockingRouterFactory
```

Comment:
Rather than override this, could you comment this statement? Ideally, we should use the defaults (which at some point will be changed to the router).